tools/updateHeaders.pl

```diff
@@ -179,6 +235,18 @@ sub extract_tarchive {
 	 return $dcmdir;
 }
 
+=pod
+
+=head3 update_file_headers($file, $setRef)
+
+Updates the headers of a DICOM file given as an argument to that function.
+
+INPUTS:
+  - $file  : DICOM file to update headers information
```

Comment:
DICOM file __in which__ to update header information

Reply:
done